events.lua

```diff
@@ -1,5 +1,8 @@
 function register_events(event)
     gauge_tick:set(game.tick)
+    gauge_connected_player_count:set(#game.connected_players)
```

Comment:
put this in a function call register_events_players

Reply:
fixed in 353e8b1